src/hooks/useEvm/useEvm.ts

```diff
@@ -1,61 +1,70 @@
-import type { ChainId } from '@shapeshiftoss/caip'
-import { fromChainId } from '@shapeshiftoss/caip'
+import { CHAIN_NAMESPACE, type ChainId, isChainReference, toChainId } from '@shapeshiftoss/caip'
 import type { ETHWallet } from '@shapeshiftoss/hdwallet-core'
 import { useCallback, useEffect, useMemo, useState } from 'react'
 import { useWallet } from 'hooks/useWallet/useWallet'
-import { bn } from 'lib/bignumber/bignumber'
 import { getSupportedEvmChainIds } from 'lib/utils/evm'
 import { selectFeatureFlags } from 'state/slices/preferencesSlice/selectors'
 import { useAppSelector } from 'state/store'
 
-const chainIdFromEthNetwork = (
-  ethNetwork: string | undefined,
+const evmChainIdFromChainId = (
+  maybeEvmChainId: ChainId | undefined,
   supportedEvmChainIds: ChainId[],
 ): ChainId | undefined => {
-  if (!ethNetwork) return
-
-  return supportedEvmChainIds.find(chainId => fromChainId(chainId).chainReference === ethNetwork)
+  if (!maybeEvmChainId) return
+  return supportedEvmChainIds.find(chainId => chainId === maybeEvmChainId)
```

Comment:
As discussed, this was previously useful as a conversion kind of util but is useless now - we have `isEvmChainId()` as a guard if we need to